app/src/main/res/values/strings.xml

```diff
@@ -235,7 +235,29 @@
     <string name="firstrun_close_button">OK, got it!</string>
     <string name="firstrun_skip_button">Skip</string>
     <string name="firstrun_next_button">Next</string>
+   
+    <!-- The welcome titles in Firefox Lite 2.5 -->
+    <string name="firstrun_fxlite_2.5_title_1">Hello!</string>
+    <!-- The welcome titles in Firefox Lite 2.5 for asking users to choose a main purpose of downloading Firefox Lite -->
+    <string name="firstrun_fxlite_2.5_title_2">What brings you to Firefox Lite?</string>
```

Comment:
`Firefox Lite` should be replaced by placeholder (`%1$s`)

Reply:
OK!